index.js

```diff
@@ -177,16 +176,16 @@ Parser.prototype._handleError = function _handleError(line) {
 
       // Need to set this value
       if (m[0] === 'actual') {
-        lastAssert.error.actual = trim(m[1]);
+        lastAssert.error.actual = m[1].trim();
```

Comment:
maybe we should make sure the values are always there before calling trim ? the following apply a bit everywhere (yet it require a recent node version so we should write it on the readme)
```suggestion
        lastAssert.error.actual = m[1]?.trim();
```

Reply:
I think publishing for more recent versions of node would be a good idea. We could add a build and packaging step to transpile for backwards compatibility. I have published what I have for now and will look to maintain and update in the future. Feel free to contribute to the fork and I'll try to keep up